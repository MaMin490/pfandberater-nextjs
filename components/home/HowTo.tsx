import React from 'react';
import { Carousel } from 'antd';
import styled from 'styled-components';

import { device } from '../../public/global/media-query';
import { theme } from '../../public/global/colors';

const { colors } = theme;

const H3 = styled.h3`
  height: 20rem;
  color: white;
  line-height: 15rem;
  text-align: center;
  background: ${colors.lightGrey};

  @media ${device.tablet} {
    height: 35rem;
  }
`;

// const IMG = styled.img`
//   height: 20rem;

//   }
// `;

export default function HowTo(): JSX.Element {
  return (
    <Carousel autoplay>
      <div>
        {/* <IMG src="/CTCLogo.png" alt="bladfewf" /> */}
        <H3>1</H3>
      </div>
      <div>
        {/* <IMG src="/how-to-2.jpg" alt="bladfewf" /> */}
        <H3>2</H3>
      </div>
      <div>
        {/* <IMG src="/How-To.png" alt="bladfewf" /> */}
        <H3>3</H3>
      </div>
    </Carousel>
  );
}
