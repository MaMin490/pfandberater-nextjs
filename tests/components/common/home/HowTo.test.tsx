import React from 'react';
import { shallow } from 'enzyme';

import HowTo from '../../../../components/home/HowTo';

describe('HowTo', () => {
  it('should render correctly', () => {
    const component = shallow(<HowTo />);

    expect(component).toMatchSnapshot();
  });
});
