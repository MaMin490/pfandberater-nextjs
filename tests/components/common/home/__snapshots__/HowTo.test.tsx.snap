// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowTo should render correctly 1`] = `
<ForwardRef
  autoplay={true}
>
  <div>
    <styled.h3>
      1
    </styled.h3>
  </div>
  <div>
    <styled.h3>
      2
    </styled.h3>
  </div>
  <div>
    <styled.h3>
      3
    </styled.h3>
  </div>
</ForwardRef>
`;
